test/IRGen/conditional-dead-strip-objc-ir.swift

```diff
@@ -6,18 +6,25 @@
 
 // RUN: %target-build-swift -Xfrontend -conditional-runtime-records %s -emit-ir -o - | %FileCheck %s
 
+import Foundation
+
 public class Class {
 }
 
+@objc public protocol ObjCAnnotatedProtocol {
+}
+
 // CHECK:      @llvm.{{(compiler.)?}}used = appending global [
 // CHECK-SAME:   @"$s4main5ClassCHn"
 // CHECK-SAME: ], section "llvm.metadata"
 
-// CHECK:      !llvm.used.conditional = !{[[M1:!.*]], [[M2:!.*]], [[O1:!.*]]}
+// CHECK:      !llvm.used.conditional = !{[[M1:!.*]], [[M2:!.*]], [[M3:!.*]], [[O1:!.*]]}
```

Comment:
The naming convention for these substitution blocks in not clear to me, not sure if the addition of `M3` and `M3A` are appropriate.